### ISSUE TITLE
Automatic suspend on battery is set to 20 minutes

### ISSUE THREAD

**Issue by brs17** [pop-os/gnome-control-center#15]
The "Automatic Suspend" feature is currently enabled by default on battery power for 20 minutes. It is desired to disable this feature out of the box.

**Comment by mmstick** [comment]
Is this only an issue in disco?

**Comment by jackpot51** [comment]
No, this is everywhere and needs to be addressed in default-settings
